runtime/j9vm/jvm.c

```diff
@@ -322,6 +322,351 @@ static BOOLEAN librariesLoaded(void);
 #define LD_ENV_PATH "LIBPATH"
 #endif /* defined(J9ZTPF) */
 
+#define J9_SIG_ERR -1
+
+#if defined(SIGHUP)
+#define J9_SIGHUP SIGHUP
+#else /* defined(SIGHUP) */
+#define J9_SIGHUP J9_SIG_ERR
+#endif /* defined(SIGHUP) */
+
+#if defined(SIGINT)
+#define J9_SIGINT SIGINT
+#else /* defined(SIGINT) */
+#define J9_SIGINT J9_SIG_ERR
+#endif /* defined(SIGINT) */
```

Comment:
Rather than defining all these `J9_SIG*` macros, I suggest the entries in the table below should be conditional on whether the corresponding signal macro is defined (mapping strings to `J9_SIG_ERR` is not helpful).

Reply:
fixed.